### PR TITLE
prov/rxd: Added missed EP attributes

### PR DESCRIPTION
**Pull request by dmitrygladkov**
Signed-off-by: Dmitry Gladkov <dmitry.gladkov@intel.com>

### CODE REVIEW

File: prov/rxd/src/rxd_attr.c [hunk 1]
```diff
@@ -63,7 +63,11 @@ struct fi_ep_attr rxd_ep_attr = {
 	.protocol_version = 1,
 	.max_msg_size = SIZE_MAX,
 	.tx_ctx_cnt = 1,
-	.rx_ctx_cnt = 1
+	.rx_ctx_cnt = 1,
+	.max_order_raw_size = SIZE_MAX,
+	.max_order_war_size = SIZE_MAX,
```
Comment (shefty) [inline prov/rxd/src/rxd_attr.c]: Is WAR ordering guaranteed?  @aingerson can you confirm the supported data ordering semantics?

**Comment by hppritcha** [comment]
bot:lanl:retest

File: prov/rxd/src/rxd_attr.c (hunk 1 above)
Comment (aingerson) [reply prov/rxd/src/rxd_attr.c]: No, WAW and RAW are guaranteed but WAR ordering is not.

Comment (dmitrygladkov) [reply prov/rxd/src/rxd_attr.c]: @shefty  @aingerson Thanks! Fixed

**Comment by shefty** [comment]
The Intel CI error appears to be some system initialization issue. AWS CI is a socket error

**Pull request merged**